setup.cfg

```diff
@@ -2,7 +2,7 @@
 max-line-length = 120
 exclude = docs,venv,env,*.egg,migrations
 max-complexity = 21
-ignore = E402,E722
+extend-ignore = E402,E722
```

Comment:
* https://stackoverflow.com/questions/68161741/python-black-formatter-conflict-with-rule-flake8-w503-in-vscode
* https://github.com/psf/black/blob/06ccb88bf2bd35a4dc5d591bb296b5b299d07323/docs/guides/using_black_with_other_tools.md#flake8